python/ray/tune/trial.py

```diff
@@ -189,9 +240,25 @@ def __init__(self,
         if trial_name_creator:
             self.custom_trial_name = trial_name_creator(self)
 
+    def init(self):
```

Comment:
the nice thing about making this specifically `init_logger` was that in various places of the codebase, we stop the trial but keep the logger. Changing this to `init` makes it harder to think about (it's asymmetric to "stop").

Reply:
ah gotcha, will refactor